src/service/extension-script.js

```diff
@@ -154,8 +154,27 @@ export function createExtensionScript(win, extensionId, version) {
     extensionId,
     version,
     getMode(win).localDev
-  );
-  scriptElement.src = scriptSrc;
+    );
+
+  if (self.trustedTypes && self.trustedTypes.createPolicy) {
+    const policy = self.trustedTypes.createPolicy(
+      'validator-integration#loadScript',
```

Comment:
Can you update the name for this use case please? Maybe something like 'extension-script#createExtensionScript'

Reply:
Yep, sorry forgot to change this.